iceoryx_posh/test/integrationtests/test_posh_mepoo.cpp

```diff
@@ -247,23 +242,42 @@ class Mepoo_IntegrationTest : public Test
         return true;
     }
 
-    void getMempoolInfoFromIntrospection(MemPoolInfoContainer& mempoolInfo)
+    void getMempoolInfoFromIntrospection(MemPoolInfoContainer& memPoolInfoContainer)
     {
-        auto currentUser = iox::posix::PosixUser::getUserOfCurrentProcess();
-        auto memoryManager = m_roudiEnv->m_roudiApp->m_mempoolIntrospection.m_segmentManager
-                                 ->getSegmentInformationWithWriteAccessForUser(currentUser.getName())
-                                 .m_memoryManager;
-        ASSERT_TRUE(memoryManager.has_value());
-        m_roudiEnv->m_roudiApp->m_mempoolIntrospection.copyMemPoolInfo(memoryManager.value().get(), mempoolInfo);
-
-        // internally, the chunks are adjusted to the additional management information;
-        // this needs to be subtracted to be able to compare to the configured sizes
-        for (auto& mempool : mempoolInfo)
+
+        iox::runtime::PoshRuntime::initRuntime(iox::roudi::INTROSPECTION_APP_NAME);
+
+        iox::popo::SubscriberOptions opts;
+        opts.queueCapacity = 1U;
+        opts.historyRequest = 1U;
+
+        iox::popo::Subscriber<iox::roudi::MemPoolIntrospectionInfoContainer> subscriber(iox::roudi::IntrospectionMempoolService, opts);
+        ASSERT_THAT(subscriber.getSubscriptionState(), iox::SubscribeState::SUBSCRIBED);
+
+        iox::popo::WaitSet<1> waitset;
```

Comment:
The difference in time is ~0.5sec for all tests in this file. `time ./build/posh/test/posh_integrationtests --gtest_filter="Mepoo_IntegrationTest*"`

With change
```
real    0m5.136s
```

W/o change
```
real    0m4.673s
```

Reply:
Should be ok for now. We'll have to see how deal with the overall time spent on CI pipelines in future.